### DISCUSSION TITLE
Sending templated email to AmazonSES

### DISCUSSION THREAD

**Discussion by gamer22026** [magicbell/community#146] [Questions]
We use templates in SES to currently send email. Want to use Magicbell to send templated  email using Amazon SES. I've added the following in the overrides section, but when testing, even though the POST shows no errors from Magicbell, nothing is sent and there are not even any logs on Magicbell that there was ever an attempt to send this to SES. Is this supposed to work? If I have an amazon_ses override of just the Source, that works. Only when trying to add the templated email stuff do I get nothing (no errors on POST, no magicbell logging, no email from SES).
```
     "overrides": {
        "providers": {
          "amazon_ses": {
            "Source": "OurService <noreply@outservice.com",
            "Destination": {
              "ToAddresses": [ "testuser@somedomain.com"
              ]
             },
            "Action": "SendTemplatedEmail",
            "Template": "ScanResult",
            "TemplateData": "{ \"title\":\"What a scan\", \"scanResultsCount\": \"500\", \"cloudfrontHost\": \"somehost\", \"executionId\": \"111111\", \"trackingId\": \"55555\", \"userId\": \"someuser\" }"
          }
        }
      }
```

**Comment by unamashana** [comment]
@gamer22026 Sorry about the issue. Are you saying that you don't see any email delivery attempts in the logs page once you start sending the Template override?

**Reply by gamer22026** [reply]
Correct. The POST to MB shows no error, but no logs from MB side exist regarding the attempt

**Reply by gamer22026** [reply]
If I send with just a simple amazon_ses override (like this for example) it works fine and I see logs.
```     "overrides": {
        "providers": {
          "amazon_ses": {
            "Source": "OurService <noreply@outservice.com"
          }
        }
      }

**Reply by gamer22026** [reply]
If I could get this working it would help us work around https://github.com/orgs/magicbell-io/discussions/145
